PORTAL/jobs/_job.py

```diff
@@ -225,7 +249,7 @@ def _build_send_script(self, queue_log, pushfsfields=None, pullfsfields=None, *,
             ('request', 'metadata'),
             ('work', 'job_script'),
             ('result', 'metadata'),
-            *pushfsfields,
+            *(pushfsfields or [])
```

Comment:
FYI, a trailing comma on the last item in a multiline compound literal can help reduce PR noise and occasional annoyance-level bugs.